util/util.go

```diff
@@ -191,13 +158,14 @@ func GetMachinesForCluster(ctx context.Context, c client.Client, cluster *cluste
 	return &machines, nil
 }
 
-// SemVerToOCIImageTag is a helper function that replaces all
+// SemverToOCIImageTag is a helper function that replaces all
 // non-allowed symbols in tag strings with underscores.
 // Image tag can only contain lowercase and uppercase letters, digits,
 // underscores, periods and dashes.
 // Current usage is for CI images where all of symbols except '+' are valid,
 // but function is for generic usage where input can't be always pre-validated.
 // Taken from k8s.io/cmd/kubeadm/app/util
+// Deprecated: Please use the functions in util/container
 func SemverToOCIImageTag(version string) string {
 	return ociTagAllowedChars.ReplaceAllString(version, "_")
```

Comment:
This should probably also call the similar function in the `util/container` package